src/jenkins/ComponentIntegTestStatus.groovy

```diff
@@ -106,6 +115,13 @@ class ComponentIntegTestStatus {
                         ]
                 ]
         ]
+        if (this.qualifier != null && this.qualifier != "None") {
```

Comment:
Too many of this calls can we replace with: https://github.com/opensearch-project/opensearch-build-libraries/blob/a0e5039b16a98ebb1347d83c4a14dc53491cec14/vars/abortStaleJenkinsJobs.groovy#L63

Reply:
We want to check for "None" as that is what is indexed by default for non-qualifiers while indexing data.

Reply:
Added a private method to incorporate above logic.
Thanks!